docs/development_guide/remote_processing.rst

```diff
@@ -50,14 +50,14 @@ SiliconCompiler also supports private servers which require authentication to ac
    "password": "your-key"
    }
 
-Use a text editor to create the credentials file. Alternatively you can use :ref:`sc-configure` app to generate it from the command line.
+Use a text editor to create the credentials file. Alternatively you can use :ref:`sc-remote` app to generate it from the command line.
 
 .. code-block:: console
 
-  (venv) sc-configure
-  Remote server address: your-server
-  Remote username: your-username
-  Remote password: your-key
+  (venv) sc-remote -configure
+  Remote server address (leave blank to use default server): your-server
+  Remote username (leave blank for no username): your-username
+  Remote password (leave blank for no password): your-key
```

Comment:
Setting the port would have been very useful for my usecase.

Reply:
you can specify the port via `your-server:your-port`